src/components/pages/Styleguide/StyleguideElements/StyleguideInputText.jsx

```diff
@@ -80,6 +80,20 @@ const StyleguideInputText = () => {
       type="text"
     />
   `
+  const inputTextWithCaracterCount = String.raw`
```

Comment:
```suggestion
  const inputTextWithCaracterCount = `
```
Et si tu pouvais l'enlever partout ça serait top.